app/(site)/alle-arrangementer/mot-en-grunder/page.tsx

```diff
@@ -0,0 +1,7 @@
+import EventPage from '@/components/events/eventPage';
+
+const Event7Page = () => {
+    return <EventPage eventId='7' />;
```

Comment:
@Hannah-Sofie Here you can change the name to MotEnGrunderPage or something descriptive. This goes for all pages.